lms/events/event.py

```diff
@@ -101,3 +102,33 @@ def _get_assignment_id(self):
             return assignment.id
 
         return None
+
+
+class AdminChangeEvent(BaseEvent):
+    def __init__(self, instance, *args, **kwargs):
+        self._instance = instance
+        super().__init__(*args, **kwargs)
+
+    def _get_data(self):
+        changes = {}
+        instance_details = inspect(self._instance)
+        for attr in instance_details.attrs:
+            history = instance_details.get_history(attr.key, True)
+
+            if not history.has_changes():
+                continue
+
+            changes[attr.key] = (1, history.added[0])
+
+        return {"user": self.request.identity.userid, "changes": changes}
```

Comment:
Added a comment around this. Its `userid` and that depends on the auth method.

For LTIUser this is irrelevant because the event itself (rather than event_data) would relate to one "User".

For the  immediate use case of this `userid` would be the email used in google's oauth.